### DISCUSSION TITLE
Report an Issue/Bug/error 🐞🐞

### DISCUSSION THREAD

**Discussion by sandeshlavshetty** [sandeshlavshetty/buss_poll#15] [Ideas]
**Expected Format**
Example :-
{
**Bug Description**:
When submitting a poll, the confirmation ticket does not generate.

**Steps to Reproduce**:
1. Log in to the app.
2. Go to the polling page.
3. Submit your preferred class timing.
4. Wait for the confirmation ticket to generate (this step fails).

**Expected Result**:
A confirmation ticket with the selected time should appear.

**Actual Result**:
No ticket appears, and an error message is shown.

**Error Message (if any)**:
"Ticket generation failed."

**Browser/Device**:
Google Chrome, Windows 10

**Supported images or screenshots**
}

**Comment by aprameyagoud** [comment]
@sandeshlavshetty  
Issue Description:
I have to login every time I open the website.

Expected Result:
Everytime I open the website, it should be logged in atleast for a few days.

Actual Result:
It asks to login everytime even if it is a matter of seconds.

Error Message (if any):
NULL

Browser/Device:
Google Chrome, phone.

Supported images or screenshots
NULL
